plugins/updater/guest-js/index.ts

```diff
@@ -70,6 +85,23 @@ class Update extends Resource {
   }
 }
 
+class DownloadedBytes extends Resource {
```

Comment:
instead of a new class, let's just store it on Update class, we will probably need to override `Resource` close method to also close the stored bytes if found.

Reply:
Ok, I was concerned about if we change how the resource is cleaned up, we'll need to update it as well, what about we store the `DownloadedBytes` resource to `Update`?

Reply:
sure, let's do `DownloadedBytes` but don't export it